catalog/ui/catalog-ui-search/src/main/java/org/codice/ddf/catalog/ui/metacard/sharing/ShareableMetacardType.java

```diff
@@ -0,0 +1,57 @@
+/**
+ * Copyright (c) Codice Foundation
+ *
+ * <p>This is free software: you can redistribute it and/or modify it under the terms of the GNU
+ * Lesser General Public License as published by the Free Software Foundation, either version 3 of
+ * the License, or any later version.
+ *
+ * <p>This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+ * without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Lesser General Public License for more details. A copy of the GNU Lesser General Public
+ * License is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ */
+package org.codice.ddf.catalog.ui.metacard.sharing;
+
+import com.google.common.collect.ImmutableSet;
+import ddf.catalog.data.AttributeDescriptor;
+import ddf.catalog.data.MetacardType;
+import ddf.catalog.data.impl.types.AssociationsAttributes;
+import ddf.catalog.data.impl.types.CoreAttributes;
+import ddf.catalog.data.impl.types.SecurityAttributes;
+import java.util.Collection;
+import java.util.Set;
+import java.util.stream.Collectors;
+
+public class ShareableMetacardType implements MetacardType {
+  private static final Set<AttributeDescriptor> ATTRIBUTE_DESCRIPTORS =
+      ImmutableSet.of(
+              new CoreAttributes(),
+              new SecurityAttributes(),
```

Comment:
❓ Is a `ShareableMetacardType` needed?  The current sharing policy for work spaces I think uses `CoreAttributes` which everything should have and `SecurityAttributes`.  So sharing could be added by adding the `SecurityAttributes` to any type.

Reply:
If I understand, previously, workspaces used `WorkspaceAttributes` and checked sharing viability through the `WORKSPACE_SHARING` attribute descriptor, which had a value of `metacard.sharing`. So I guess Ideally, workspaces should be getting rid of that too, no?

Reply:
I think we can avoid a shareable metacard type. We already have some sharing with `security.access*`, all that's missing is the concept of governance or 'administrator privileges'.